examples.md

```diff
@@ -52,7 +52,41 @@ network:
           addresses: [10.10.10.1, 1.1.1.1]
 ```
 
-## Wireless interfaces
+## Connecting multiple interfaces with DHCP
+
+Many systems now include more than one network interface. Servers will commonly need to connect to multiple networks, and may require that traffic to the Internet goes through a specific interface despite all of them providing a valid gateway.
+
+One can achieve the exact routing desired over DHCP by specifying a metric for the routes retrieved over DHCP, which will ensure some routes are preferred over others. In this example, 'enred' is preferred over 'engreen', as it has a lower route metric:
+
+```yaml
+network:
+  version: 2
+  ethernets:
+    enred:
+      dhcp4: yes
+      dhcp4-overrides:
+        route-metric: 100
+    engreen:
+      dhcp4: yes
+      dhcp4-overrides:
+        route-metric: 200
+```
+
+## Connecting to an open wireless network
+
+Netplan easily supports connecting to an open wireless network (one that is not secured by a password), only requiring that the access-point is defined:
```

Comment:
Is access-point usually a hyphenated word?  Additionally, in the previous example you call out enred and engreen as being interfaces but "opennetwork", used below, isn't so it may be confused as something else.